pkgs/applications/version-management/git-and-tools/git-annex-remote-googledrive/default.nix

```diff
@@ -0,0 +1,22 @@
+#{ stdenv, pythonPackages, fetchPypi, rclone, makeWrapper }:
+{ stdenv, lib, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "git-annex-remote-googledrive";
+  version = "0.11.1";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "1vaycxmxhx8g797nzrnkmz5vbmjy4x7x3b7a2xb7yz6grgcxqgld";
+  };
```

Comment:
By convention we also add a 

```
# no tests
doCheck = false;
```

remark if a python package comes without tests.